_posts/2025-04-29-juan-luis-opensouce-journey.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title:  "A Journey Through Open Source: From Hacker Culture to the Age of AI"
+description: "A Journey Through Open Source: From Hacker Culture to the Age of AI"
+date:   2025-04-29 10:00:35 +0300
+image:  '/images/blogs/interview_sam_miyamoto.png'
+tags:   [opensource, community]
```

Comment:
```suggestion
image:  '/images/blogs/blog-placeholder.png'
tags:   [opensource, community]
```

Reply:
@SangamSwadiK I create a placeholder for images, so the preview blogs would not confuse reviewers.